newswires/client/src/SideNav.tsx

```diff
@@ -134,6 +159,38 @@ export const SideNav = () => {
 		];
 	}, [activePreset, config.query, handleEnterQuery]);
 
+	const supplierItems: EuiPinnableListGroupItemProps[] = suppliers.map(
+		({ label, colour, isActive, onClick, onTickerClick }) => ({
+			id: label,
+			label,
+			onClick,
+			isActive,
+			iconType: () => (
+				<div
+					css={css`
+						width: 0.5rem;
+						height: 1.5rem;
+
+						margin-right: 12px;
+						background-color: ${isActive ? colour : 'transparent'};
+					`}
+				/>
+			),
+			color: isActive ? 'primary' : 'subdued',
+			pinnable: false,
+			extraAction: {
```

Comment:
generally this looks great! but when hovering over a non-selected supplier the contrast between the popout button and the supplier button is very faint -- could we make the contrast easier to see?
<img width="280" alt="image" src="https://github.com/user-attachments/assets/313b9b96-9a36-4578-904b-78febc9d2900" />